packages/core/src/loop/workflows/agentic-execution/llm-mapping-step.ts

```diff
@@ -1,38 +1,18 @@
 import type { ToolSet } from 'ai-v5';
```

Comment:
This file wasn't renamed, it was created, GitHub is confused. This step I just extracted from the agentic-execution workflow. It has 2 other steps extracted into separate files, but the third step was created in file. So I moved it out for consistency and readability